### ISSUE TITLE
[FR]: Not able to get opik-optimizer version >0.7.1

### ISSUE THREAD

**Issue by asjad895** [comet-ml/opik#2537]
### Proposal summary

As we are installing using pip u can see version

![Image](https://github.com/user-attachments/assets/a1ae4ebc-dc34-4dec-9c93-f546b40f313c)
which causing conflict in docs and librray.
Also i need help to create correct config if using 0.7.1 version
Is this metric correctly mapped

![Image](https://github.com/user-attachments/assets/9a7161c4-16bd-4850-af88-4aa196b3936a)

is this setup is correct
`def get_optimizer_config(self) -> Dict[str, Any]:
        """
        Get the current optimizer configuration.
        
        Returns:
            Dictionary containing optimizer configuration
        """
        task_config = TaskConfig(
            instruction_prompt = self.system_prompt,
            use_chat_prompt=True,
            input_dataset_fields = ['user_request'],
            output_dataset_field = 'trigger_response',
        )
        metric_config = MetricConfig(
            metric = TriggerEvaluator(name="trigger_evaluator", track=True),
            inputs = {
                "text_request": "user_request"
            }
        )
        return {
            "model": self.model,
            "temperature": self.temperature,
            "max_tokens": self.max_tokens,
            "max_rounds": self.max_rounds,
            "num_prompts_per_round": self.num_prompts_per_round,
            "num_threads": self.num_threads,
            "seed": self.seed,
            "dataset": self.training_dataset,
            "metric_config": metric_config,
            "task_config": task_config
        }`

also uploading notebook for full context:

**Comment by asjad895** [comment]
Each dataset item format.
`
{
    "user_request": "Send a recommendation email when user purchases a product",
    "trigger_response": {
        "trigger_id": "purchase_trigger",
        "type": "event",
        "trigger_name": "purchase_trigger",
        "description": "user wants to trigger a journey when a user purchases a product means they have completed the purchase",
        "parameters": {
            "channel_type": "email",
            "event_type": null
        },
        "conditions": [
            {
                "type": "segment",
                "parameters": {
                    "segmentation_description": "Users who have purchased a product",
                    "condition_id": "cond_001",
                    "channel_type": null,
                    "event_type": null,
                    "on_true": "purchase_trigger",
                    "on_false": null
                }
            }
        ]
    }
}`

**Comment by dsblank** [comment]
Can you try: `pip install opik_optimizer --upgrade` and check out the latest version:

https://www.comet.com/docs/opik/agent_optimization/opik_optimizer/quickstart
